onnx/onnxifi.h

```diff
@@ -127,9 +127,10 @@ typedef uint64_t onnxPointer;
 #define ONNXIFI_STATUS_UNSUPPORTED_PARAMETER 0x0022
 #define ONNXIFI_STATUS_UNSUPPORTED_SHAPE 0x0023
 #define ONNXIFI_STATUS_UNSUPPORTED_DATATYPE 0x0024
-#define ONNXIFI_STATUS_UNIDENTIFIED_NAME 0x0025
-#define ONNXIFI_STATUS_MISMATCHING_SHAPE 0x0026
-#define ONNXIFI_STATUS_MISMATCHING_DATATYPE 0x0027
+#define ONNXIFI_STATUS_UNSUPPORTED_TAG 0x0025
```

Comment:
What's reason to reuse 0x0025 here while skipping 0x0026 and 0x0027?

Reply:
To add space for `ONNXIFI_STATUS_UNSUPPORTED_something` codes I'll add later

Reply:
Good point! Added more space in there